app/views/users/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2> Edit your profile </h2>
+
+<%# <form action="/users" method="post" accept-charset="UTF-8"> %>
+  <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %><%#%"> %>
+  <%# <label for="username">Username</label>
+  <input type="text" name="user[username]" id="username" placeholder="username">
+  <label for="email">Email</label>
+  <input type="email" name="user[email]" id="email" placeholder="email">
+  <label for="password">username</label>
+  <input type="password" name="user[password]" id="password" placeholder="password">
+  <input type="submit" value="submit">
+</form> %>
+
+<%# <%= form_tag('/users', method: "post") do %> <%#%%>
+  <%# <%= label_tag(:username, "username") %> <%#%>
+  <%# <%= text_field_tag(:username) %> <%#%>
+  <%# <%= label_tag(:email, "email") %> <%#%>
+  <%# <%= email_field_tag(:email) %> <%#%>
+  <%# <%= label_tag(:password, "password") %> <%#%>
+  <%# <%= password_field_tag(:password) %> <%#%>
+  <%# <%= submit_tag("create user") %> <%#%>
+<%# <% end %> <%#%>
+
+<%= form_for @user do |f|%>
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+  <%= f.label :email %>
+  <%= f.email_field :email %>
+  <%= f.label :password %>
+  <%= f.password_field :email %>
```

Comment:
this should be ` <%= f.password_field  :password %>`

Reply:
## We worked on all the changes below

- we replaced the redirect_to with render :new after failing to create a user
- we replaced the action in the forms with user_path instead of "/users"
- we replace create_params with user_params
- we added the only to the routes resources for specific actions in the controller
- we replaced the email on password field with password
- we also added validations for password